DuckDuckGo/OnboardingExperiment/Background/OnboardingGradient.swift

```diff
@@ -23,57 +23,22 @@ import Onboarding
 struct OnboardingGradientView: View {
     @Environment(\.colorScheme) private var colorScheme
 
-    private let type: OnboardingGradientType
-
-    init(type: OnboardingGradientType) {
-        self.type = type
-    }
-
     var body: some View {
-        switch (type, colorScheme) {
-        case (.default, .light):
-            linearLightGradient
-        case (.default, .dark):
-            linearDarkGradient
-        case (.highlights, _):
-            // If highlights experiment use new common gradient for iOS and macOS
+        switch colorScheme {
+        case .dark:
             OnboardingGradient()
+        case .light:
+            // iOS 15 doesn't render properly the light EllipticalGradient while the Dark gradient is rendered correctly
```

Comment:
Issue explained in https://app.asana.com/0/0/1208839072951158/1208895765286109/f